.asf.yaml

```
@@ -0,0 +1,39 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# https://cwiki.apache.org/confluence/display/INFRA/git+-+.asf.yaml+features
+
+github:
+  description: "Apache Accumulo Access Control Library"
+  homepage: https://accumulo.apache.org
+  labels:
+    - accumulo
+    - big-data
+    - hacktoberfest
+  features:
+    wiki: false
+    issues: true
+    projects: false
```

It's useful to use projects. In the main repo, we use projects to track milestones for multiple versions still being developed (since GitHub doesn't support tagging multiple milestones). We probably won't use that here... we can keep one linear set of releases and just use the normal milestone feature. However, we also use projects for other things, and that might end up being useful in this repo as well. I would go ahead and enable it since it's not hurting anything to have it enabled and there's a good chance we'll end up using it.

```suggestion
    projects: true
```